pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1222,7 +1222,7 @@ class Timedelta(_Timedelta):
         'm', 'minute', 'min', 'minutes', 'T', 'S', 'seconds', 'sec', 'second',
         'ms', 'milliseconds', 'millisecond', 'milli', 'millis', 'L',
         'us', 'microseconds', 'microsecond', 'micro', 'micros', 'U',
-        'ns', 'nanoseconds', 'nano', 'nanos', 'nanosecond', 'N'}
+        'ns', 'nanoseconds', 'nano', 'nanos', 'nanosecond', 'N'}.
```

Comment:
Not sure about the period here.  This is a sufficiently special case I imagine we don't have a convention

Reply:
technically, per the [contributing_docstring#parameter-types](https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html#parameter-types), I think it should be 

```
Parameters
----------
unit : {'Y', 'M', 'W', 'D', 'days', 'day', 'hours', hour', 'hr', 'h', 'm', 'minute', 'min', 'minutes', 'T', 'S', 'seconds', 'sec', \
'second', 'ms', 'milliseconds', 'millisecond', 'milli', 'millis', 'L', 'us', 'microseconds', 'microsecond', 'micro', 'micros', \
'U', 'ns', 'nanoseconds', 'nano', 'nanos', 'nanosecond', 'N'}, default 'ns'
    Denote the unit of the input, if input is an integer.
```

but I wasn't sure, so I just added a period. I can change it to the above, if thats agreeable.

Reply:
I think you're right, but it is a bit akward because it is a long list.  Let's ask @datapythonista for an opinion

Reply:
My preference would be to add to the guidelines that if the set of values is too long for a line, we add the type, and then we add the possible values in a list. Something like:
```
Parameters
----------
unit : str
    Denote the unit of the input, if input is an integer.

    Possible values:
    - 'Y': years
    - 'M': months
    - ...
```
I think that'll be much easier to read, both in the code and in the rendered version.

Reply:
@datapythonista updated. ill submit a separate PR for updating the guidelines once all the doc errors have been fixed. i have a feeling there will be a few more edge cases without conventions